backend/src/asiakirja/adapter/commonKutsuAdapter.ts

```diff
@@ -209,7 +209,7 @@ export class CommonKutsuAdapter {
 
     const isElyOrganisaatio = !!organisaatioIsEly(projektiPaallikko.organisaatio) && !!projektiPaallikko.elyOrganisaatio;
 
-    return isElyOrganisaatio ? translate(`viranomainen.${projektiPaallikko.elyOrganisaatio}`, this.kieli) : projektiPaallikko.organisaatio;
+    return isElyOrganisaatio ? translate(`viranomainen.${projektiPaallikko.elyOrganisaatio}`, this.kieli) : translate("viranomainen.VAYLAVIRASTO", this.kieli);
```

Comment:
Voikohan näin tehdä. Onko projektipäällikkö aina Väylältä?

Reply:
Ymmärrettävämmältähän tämä sinun muutos vaikuttaa kuin tuo alkuperäinen.